tests/unit/nupic/frameworks/opf/clamodel_test.py

```diff
@@ -82,7 +83,103 @@ def testTemporalAnomalyModelFactory(self):
     Temporal Anomaly configuration will return a model that can return
     inferences
     """
-    modelConfig = (
+    inferenceArgs = {u'inputPredictedField': u'auto',
+                     u'predictedField': u'c1',
+                     u'predictionSteps': [1]}
+
+    data = [
+      {'_category': [None],
+       '_reset': 0,
+       '_sequenceId': 0,
+       '_timestamp': datetime.datetime(2013, 12, 5, 0, 0),
+       '_timestampRecordIdx': None,
+       u'c0': datetime.datetime(2013, 12, 5, 0, 0),
```

Comment:
these don't need to be unicode, just use double quote strings everywhere for consistency